<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiberOutputForm</class>
 <widget class="QWidget" name="FiberOutputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="DataSelectionHLayout">
     <item>
      <layout class="QVBoxLayout" name="DataSelectionLabelVLayout">
       <item>
        <widget class="QLabel" name="DTIDataLabel">
         <property name="text">
          <string>DTI Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="EigenDataLabel">
         <property name="text">
          <string>Eigensystem Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="DataSelectionComboVLayout">
       <item>
        <widget class="QComboBox" name="DTIDataCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="EigenDataCombo"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="DataSourceGroup">
     <property name="title">
      <string>Data Source</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="DataSourceROIRadio">
        <property name="text">
         <string>Regions of Interest (Seed Points)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="DataSourceFibersRadio">
        <property name="text">
         <string>Fibers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="OutputOptionsGroup">
     <property name="title">
      <string>Output Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="OutputOptionsPerVoxelCheck">
        <property name="text">
         <string>Data per Voxel</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="OutputOptionsMeanVarCheck">
        <property name="text">
         <string>Means and Variances</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="OutputOptionsTensorCheck">
        <property name="text">
         <string>Tensor Values</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="OutputOptionsEigenCheck">
        <property name="text">
         <string>Eigenvectors</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="OutputOptionsLengthCheck">
        <property name="text">
         <string>Fiber Length</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="OutputOptionsVolumeCheck">
        <property name="text">
         <string>Fiber Volume</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="FibersHLayout">
     <item>
      <widget class="QLabel" name="FibersLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fibers:   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="FibersCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="ROILabel">
     <property name="text">
      <string>Regions of Interest: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="ROIList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="MeasuresLabel">
     <property name="text">
      <string>Scalar Measures:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="MeasuresList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="OutputHLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="OutputTXTButton">
       <property name="text">
        <string>Write Output (TXT)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OutputXMLButton">
       <property name="text">
        <string>Write Output (XML)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
